ArduPlane/Log.cpp

```diff
@@ -303,15 +306,17 @@ const struct LogStructure Plane::log_structure[] = {
 // @Field: NavPitch: desired pitch
 // @Field: Pitch: achieved pitch
 // @Field: ThO: scaled output throttle
-// @Field: RdrOut: scaled output rudder
+// @Field: RdrO: scaled output rudder
 // @Field: ThD: demanded speed-height-controller throttle
 // @Field: As: airspeed estimate (or measurement if airspeed sensor healthy and ARSPD_USE>0)
 // @Field: SAs: DCM's airspeed estimate, NaN if not available
+// @Field: AT: airspeed type ( old estimate or source of new estimate)
+// @FieldValueEnum: AT: AP_AHRS::AirspeedEstimateType
 // @Field: E2T: equivalent to true airspeed ratio
 // @Field: GU: groundspeed undershoot when flying with minimum groundspeed
 
     { LOG_CTUN_MSG, sizeof(log_Control_Tuning),     
-      "CTUN", "Qccccffffffi",    "TimeUS,NavRoll,Roll,NavPitch,Pitch,ThO,RdrOut,ThD,As,SAs,E2T,GU", "sdddd---nn-n", "FBBBB---00-B" , true },
+      "CTUN", "QccccffffBffi",    "TimeUS,NavRoll,Roll,NavPitch,Pitch,ThO,RdrO,ThD,As,AT,SAs,E2T,GU", "sdddd---n-n-n", "FBBBB---000-B" , true },
```

Comment:
I'd like to re-up my suggestion - I really do think "AsT" would be easier to understand when reviewing logs than "AT" - since all the other airspeed values are coded as "As" - will the one extra character fit?

Reply:
no room....we'd have to mod the pitch/roll....and that was not desired...only the rudder since its not used in the canned mavexplorer graphing

Reply:
RdO ?

Reply:
that works, if @tridge is okay with it